server/src/lib/db.ts

```diff
@@ -1,4 +1,4 @@
-import { MongoClient, ObjectId } from 'mongodb';
+import { MongoClient, ObjectID, ObjectId } from 'mongodb';
```

Comment:
Why do we need two different ObjectId/ObjectID imports? Surely we should be able to just use one of them and stick with that.

Reply:
Ehmm, yeah this looks odd. I did not notice the extra import, VSCode just prompted me with an auto-import, so I imported that one. I will change it to use the other one

Reply:
Just make sure it still works 🙂